gravitee-apim-console-webui/src/management/api/list/api-list.component.html

```diff
@@ -65,7 +65,7 @@ <h1>APIs</h1>
           svgIcon="gio:play-circle"
         ></mat-icon>
         <mat-icon
-          *ngIf="element.state !== 'STARTED'"
+          *ngIf="element.state && element.state !== 'STARTED'"
```

Comment:
This could be encapsulated into a single function `!isElementStarted()`. Not a must just a suggestion.

Reply:
I try to avoid functions in the html because Angular will call them continuously. With a logic statement like this in the html, it is re-evaluated only if the value of `element.state` changes.